vault/expiration.go

```diff
@@ -836,16 +851,12 @@ func (m *ExpirationManager) RevokeByToken(ctx context.Context, te *logical.Token
 		if le != nil {
 			le.ExpireTime = time.Now()
 
-			{
-				m.pendingLock.Lock()
-				if err := m.persistEntry(ctx, le); err != nil {
-					m.pendingLock.Unlock()
-					return err
-				}
-
-				m.updatePendingInternal(le)
+			// TODO: storage race with concurrent renew
```

Comment:
are these possible races what we discussed on Tues or Weds? What did we decide on for a solution?

Reply:
The solution is "leave them in".

The locking change here means that it's possible the in-memory version (in pending) and the storage version of the leaseEntry could be inconsistent.

But, it was already possible for the renew call to re-create a leaseEntry that was just deleted.